lib/ab/assigned_test.rb

```diff
@@ -46,6 +46,10 @@ def variants
       @test.variants.map(&:name)
     end
 
+    def name
```

Comment:
Could be private.
Could also be removed, and something like `@test.try(:name)` be used in `MissingVariant`, though that seems like peeking into internal a bit too much?